apko-build/action.yaml

```diff
@@ -56,7 +62,7 @@ outputs:
 
 runs:
   using: docker
-  image: "docker://ghcr.io/chainguard-dev/apko:v0.3.3"
+  image: "docker://ghcr.io/jdolitsky/apko:v9.9.9"
```

Comment:
need to swap this out after an apko release

Reply:
can't we use docker://ghcr.io/chainguard-dev/apko:canary now?

Reply:
yes, but also on canary this flags was swapped out with `--use-docker-mediatypes`, making this PR sort of moot.

We could re-do this PR to allow the option to push Docker types, but otherwise OK to close

Reply:
using canary now